index.js

```diff
@@ -1,69 +1,83 @@
-const todoInput = document.getElementById('todoInput');
-const todoList = document.getElementById('todoList');
+(function () {
+  const todoInput = document.getElementById('todoInput');
+  const todoList = document.getElementById('todoList');
 
-const doneElement = (el) => { //
-  el.classList.toggle("done");
-  const changeEl = el.parentNode.id;
-  toDoList.toggleTask(changeEl);
-}
+  const listItemForCopy = document.querySelector('.for-copy').cloneNode(true);
 
-const clickDelete = (el) => {
-  const deteledId = el.parentNode.id;
-  toDoList.deleteTask(deteledId);
-  renderElements();
-}
+  document.querySelector('.for-copy').remove();
```

Comment:
those actions are executed immediately and it's better to put them into one function. For example, init()